generate-offline-map-using-workmanager/README.md

```diff
@@ -0,0 +1,66 @@
+# Generate offline map using workmanager
```

Comment:
@shubham7109 Can the title and the project name be different? Or would you suggest changing the project/directory name too?

Reply:
I would suggest changing the module project and directory names as well, since there are scripts that require the names to be consistent. I realize this would require many changes but it would be necessary.

Reply:
The `Jetpack's` apostrophe is causing issues in multiple places. Shall I omit that?

Reply:
Sure feel free to omit it. Also, I think it would be a good idea updating the description:
>Take a web map offline as a persistent background task using Android Jetpack's [WorkManager](https://developer.android.com/topic/libraries/architecture/workmanager)